README.md

```diff
@@ -2,6 +2,20 @@
 
 Hypha Organizational Handbook (H₂O) describes the vision, processes, and culture of Hypha Worker Co-operative. 🌿🍄
 
+### About This Handbook
+<!-- Derived from: https://handbook.enspiral.com/#about-this-handbook -->
+
+This Handbook's primary audience is Hypha worker-members and their
```

Comment:
```suggestion
This Handbook's primary audience is Hypha member-owners and their
```

Reply:
We've approved language which calls ourselves "member-owners" at a previous meeitng

Reply:
I'm easy, but tried to search our notes, and only found this, where the notes didn't reflect that _per se_: https://github.com/hyphacoop/organizing/blob/master/2019-09-04-all-hands-meeting.md

Something else you were thinking? Or were notes missing?

Reply:
Pretty sure we discussed this when we finalized membership classes (https://github.com/hyphacoop/organizing/blob/master/2019-05-27-all-hands-meeting.md), but I don't see it in notes. We updated the business plan to "member-owner" afterwards and issues use that language too....

Reply:
don't recall deciding such a long time ago, thought it was much more recent, but glad to have one or the other decided :) let's get this in glossary in near future!